datafusion/physical-expr/src/expressions/column.rs

```diff
@@ -100,49 +100,3 @@ impl PartialEq<dyn Any> for UnKnownColumn {
         false
     }
 }
-
```

Comment:
It seems like `datafusion/physical-expr/src/expressions/column.rs` could also be renamed to `datafusion/physical-expr/src/expressions/unknown_column.rs` or `datafusion/physical-expr/src/expressions/unknown.rs` to better match its contents

Reply:
Renamed it to `unknown_column.rs` in https://github.com/apache/datafusion/pull/11573/commits/f4407ac092ee639ea63dd807cae0f68f49ec0d62